.travis.yml

```diff
@@ -17,6 +17,10 @@ matrix:
       python: 3.5
     - env: TOXENV=py36
       python: 3.6
+    - env: TOXENV=py37
+      dist: xenial
+      sudo: required
+      python: 3.7
```

Comment:
Should we also test against `nightly` on xenial too and allow for it to fail? Maybe something for another PR

Reply:
That would be nice, but let's leave it for another PR.